azure/functions/_abc.py

```diff
@@ -421,3 +424,88 @@ class OrchestrationContext(abc.ABC):
     @abc.abstractmethod
     def body(self) -> str:
         pass
+
```

Comment:
Rename to AuthenticationEventRequestStatus Or see if you can move it into the IEventRequest class instead.